010-anonymous-login.md

```diff
@@ -0,0 +1,101 @@
+# Anonymous Login
+
+- Implementation Owner: @TorstenDittmann
+- Start Date: 15-02-2021
+- Target Date: 16-02-2021
+- Appwrite Issue: https://github.com/appwrite/appwrite/discussions/907
+
+## Summary
+
+[summary]: #summary
+
+This RFC will introduce authentication of anonymous users in Appwrite. These temporary anonymous users can be used to allow users who haven't yet signed up to an app to work with data protected by Appwrite's permission and security rules. If an anonymous user decides to sign up to an app, they can convert their account to an E-Mail or OAuth2 authenticated one, so that they can continue to work with their protected data in future sessions.
+
+## Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+**What problem are you trying to solve?**
+
+When you develop an application, there might be times when you want to let a user interact with pieces of Appwrite before they're signed in. This also increases the converison rate of applications, since the hurdle of registering is very high.
```

Comment:
there might be times when you want to let a user interact with pieces of your application before they're signed in